src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUnitTest.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.hellbender.tools.funcotator;
+
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+
+/**
+ * Unit tests for the {@link org.broadinstitute.hellbender.tools.funcotator}.
+ * Created by jonn on 8/22/17.
+ */
+public class FuncotatorUnitTest extends GATKBaseTest {
```

Comment:
Was this class filled in in a later PR?

Reply:
I don't think it's filled out yet, but there are integration tests that check all the methods in Funcotator.

I'll fill this out when we collapse the rest of the branches.